src/pages/reader/epub.vue

```diff
@@ -116,10 +123,16 @@ export default {
     this.initRendition();
   },
   methods: {
-    initRendition() {
+    async initRendition() {
+      this.isLoading = true;
       const encodedUrl = encodeURIComponent(this.fileSrc);
       const corsUrl = `https://pdf-cors-ufdrogmd2q-uw.a.run.app/pdf-cors?url=${encodedUrl}`;
-      this.book = Epub(corsUrl);
+      const buffer = await this.$axios.$get(corsUrl, {
+        responseType: 'arraybuffer',
+      });
```

Comment:
Should we add try-catch here?

Reply:
If any error is thrown here, the whole reader fail to function
Maybe we should show some error message

Reply:
It is acceptable if it shows an error page with the raw message

Reply:
added `this.$nuxt.error`